bcipy/gui/mode/RSVPKeyboard.py

```diff
@@ -15,7 +16,10 @@
 class RSVPKeyboard(BCIGui):
     """GUI for launching the RSVP tasks."""
     event_started = False
-    PARAMETER_LOCATION = 'bcipy/parameters/parameters.json'
+
+    def __init__(self, title, size, background_color, default_param_location='bcipy/parameters/parameters.json'):
+        super(RSVPKeyboard, self).__init__(title, size, background_color)
+        self.PARAMETER_LOCATION = default_param_location
```

Comment:
The parameter default_param_location should just be renamed to param_location. Also, there are several places in the code where this location is used as a default value. This makes it error-prone if we ever need to change this location. We should just have a single place where the default value is specified and pass in the parameter for every other method. 

Also, self.PARAMETER_LOCATION shouldn't be capitalized since it's not a constant.